autoPyTorch/data/base_feature_validator.py

```diff
@@ -152,4 +154,30 @@ def transform(
             np.ndarray:
                 The transformed array
         """
+
+        raise NotImplementedError()
+
+    def list_to_dataframe(
```

Comment:
we dont need to have this on the base class as it will only be used for the tabular data right?

Reply:
It is referenced in ` fit`. It does not look nice when you reference a method that a subclass has and not the class itself.

Reply:
oh I see, thanks for this then

Reply:
I did think about the fact that `list_to_dataframe ` and where it should be placed in the future, but I thought that belongs better in another PR that does consider more than the tabular domain.

Reply:
yeah sure